src/main/java/nl/thehyve/ocdu/controllers/UploadSessionController.java

```diff
@@ -0,0 +1,54 @@
+package nl.thehyve.ocdu.controllers;
+
+import nl.thehyve.ocdu.models.UploadSession;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.*;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+/**
+ * Created by bo on 4/26/16.
+ */
+@RestController
+public class UploadSessionController {
+
+    @RequestMapping(value="/username", method= RequestMethod.GET)
+    public String username() {
+        Object principal = SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+        String username = "???";
+        if (principal instanceof UserDetails) {
+            username = ((UserDetails)principal).getUsername();
+        } else {
+            username = principal.toString();
+        }
+        return username;
+    }
+
+    @RequestMapping(value="/unfinished-sessions", method= RequestMethod.GET)
+    public List<UploadSession> unfinished_sessions() {
+
+        List<UploadSession> sessions = new ArrayList<UploadSession>();
+        Calendar c = Calendar.getInstance();
+        c.set(2015, Calendar.JUNE, 3);
+        sessions.add(new UploadSession("session_1", UploadSession.Step.MAPPING, c.getTime()));
+        c.set(2015, Calendar.JULY, 30);
+        sessions.add(new UploadSession("session_2", UploadSession.Step.MAPPING, c.getTime()));
+        c.set(2015, Calendar.AUGUST, 15);
+        sessions.add(new UploadSession("session_3", UploadSession.Step.MAPPING, c.getTime()));
+        c.set(2015, Calendar.SEPTEMBER, 15);
+        sessions.add(new UploadSession("session_4", UploadSession.Step.PATIENTS, c.getTime()));
+        c.set(2015, Calendar.OCTOBER, 15);
+        sessions.add(new UploadSession("session_5", UploadSession.Step.PATIENTS, c.getTime()));
+        c.set(2015, Calendar.NOVEMBER, 15);
+        sessions.add(new UploadSession("session_6", UploadSession.Step.EVENTS, c.getTime()));
+        c.set(2015, Calendar.DECEMBER, 18);
+        sessions.add(new UploadSession("session_7", UploadSession.Step.EVENTS, c.getTime()));
+        c.set(2016, Calendar.JANUARY, 1);
+        sessions.add(new UploadSession("session_8", UploadSession.Step.OVERVIEW, c.getTime()));
+
+        return sessions;
```

Comment:
I see that it is a test dataset. While it is good for evaluating some new designs quickly, do you think this is necessary here?